src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -262,17 +262,14 @@ def test_graph_group_scenario(self):
         account_info = self.cmd('account show').get_output_in_json()
```

Comment:
`user.name` from `az account show` may be different from the real **user principal name** if the user is a guest account in this tenant. To get the real user principal name, look for `userPrincipalName` from `az ad signed-in-user show`. 

For example my `user.name` from `az account show` is `myid@microsoft.com`, but `userPrincipalName` from `az ad signed-in-user show` is `myid_microsoft.com#EXT#@AzureSDKTeam.onmicrosoft.com`

Reply:
updated